uktrade/data-hub-frontend
# Event type filter

## Description of change

Adds an event type filter to the new react event collection page

## Test instructions

It should now be possible to filter by event type - the event types should be populated as a checkbox group. Event type chips should appear above and the url should include the selected event_types

## Screenshots

![Screenshot from 2021-06-25 11-48-30](https://user-images.githubusercontent.com/1234577/123414049-50ffd700-d5ab-11eb-9c95-23dba113e359.png)

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
